cmd/istio-cni/redirect.go

```diff
@@ -64,6 +63,11 @@ var (
 	}
 )
 
+// Tables defines interface which must be implemented by tables mechanism driver (ex. ip tables )
+type Tables interface {
+	program(string, *Redirect) error
```

Comment:
Add a documentation comment for this method. And choose a more expressive name that reflects what this is supposed to do.

Reply:
It expresses exactly what it does, it PROGRAMS, if you ask what it programs, it programs Tables.

Reply:
done, let me know if added comments are sufficient, thank you.